tkg/client/cluster.go

```diff
@@ -160,7 +160,13 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 			return false, errors.Wrap(err, "unable to get cluster configuration")
 		}
 
-		if !c.IsFeatureActivated(constants.FeatureFlagAllowLegacyCluster) {
+		allowLegacyCluster, err := c.TKGConfigReaderWriter().Get(constants.ConfigVariableAllowLegacyCluster)
```

Comment:
Can you add more context in the commit message? https://cbea.ms/git-commit/

Reply:
@DanielXiao , add more explanations into commit message, pls check it again, thanks!